.factory/automation.yml

```diff
@@ -59,6 +59,16 @@ build:
         export DEPLOY_MAVEN_USERNAME=$REPO_TYPEDB_USERNAME
         export DEPLOY_MAVEN_PASSWORD=$REPO_TYPEDB_PASSWORD
         bazel run --define version=$(git rev-parse HEAD) //:deploy-maven -- snapshot
+        bazel run --define version=$(git rev-parse HEAD) //test:deploy-maven -- snapshot
```

Comment:
This stray line seems to have slipped in! We don't have this target in this repository?